sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -11600,4 +11600,20 @@ ALTER TABLE externalmessage_history ADD COLUMN caze_id bigint;
 
 INSERT INTO schema_version (version_number, comment) VALUES (467, '[DEMIS2SORMAS] Introduce processing for physician reports #8980');
 
+-- 2022-06-13 lab organization ids #8949
+ALTER TABLE labmessage ALTER COLUMN labexternalid TYPE VARCHAR(255)[] USING ARRAY[labexternalid];
+ALTER TABLE labmessage_history ALTER COLUMN labexternalid TYPE VARCHAR(255)[] USING ARRAY[labexternalid];
+
+ALTER TABLE labmessage RENAME COLUMN labexternalid TO labexternalids;
+ALTER TABLE labmessage_history RENAME COLUMN labexternalid TO labexternalids;
```

Comment:
Those commands don't work (anymore), because the labmessage tables got renamed to externalmessage.

Reply:
fixed